packages/cli/src/index.ts

```diff
@@ -66,7 +67,11 @@ if (!process.stdin.isTTY) {
   });
   process.stdin.on('end', () => run(JSON.parse(input), program as any));
 } else if (fileName) {
-  const schema: string = readFile(fileName);
+  let schema: string = '';
```

Comment:
perhaps we should throw all of this into a function in util - or maybe add the functionality to readFile?